lib/core/src/Cardano/Wallet.hs

```diff
@@ -2261,7 +2261,6 @@ data ErrStartTimeLaterThanEndTime = ErrStartTimeLaterThanEndTime
 data ErrSelectForDelegation
     = ErrSelectForDelegationNoSuchWallet ErrNoSuchWallet
     | ErrSelectForDelegationFee ErrAdjustForFee
-    | ErrSelectForDelegationUnableToAssignInputs ErrNoSuchWallet
```

Comment:
:+1: I guess that's also the case for `ErrJoinStakePoolUnableToAssignInputs` and `ErrQuitStakePoolUnableToAssignInputs`.